### PR TITLE
Fix eukulele

### PR DESCRIPTION
**Pull request by danilodileo**
## PR checklist

Eukulele it's working again. I deleted the mvdir.nf and tested the pipeline with all features. 
I also updated the nextflow_schema.json

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
                     "pattern": "^\\S+\\.csv$",
                     "schema": "assets/schema_input.json",
                     "description": "Path to comma-separated file containing information about the samples in the experiment.",
-                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row. See [usage docs](https://nf-co.re/metatdenovo/usage#samplesheet-input).",
+                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row. See [usage docs](https://nf-co.re//usage#samplesheet-input).",
```
Comment (emnilsson) [inline nextflow_schema.json]: We don't have a usage-page, maybe leave the url empty? Or add in metatdenovo again?

File: nextflow_schema.json [hunk 2]
```diff
@@ -42,72 +42,98 @@
...
                     "type": "integer",
-                    "description": "Kmer length in first call of normalize-by-median.py, default 20.",
-                    "hidden": true
+                    "default": 20,
+                    "description": "Kmer length in first call of normalize-by-median.py, default 20."
+                },
+                "assembler": {
+                    "type": "string",
+                    "default": "megahit",
+                    "description": "Decide which assembler to use"
```
Comment (emnilsson) [inline nextflow_schema.json]: I vote adding the options and stating default, as for orf_caller.

File: nextflow_schema.json [hunk 3]
```diff
@@ -42,72 +42,98 @@
...
-            "default": "",
-            "properties": {
-                "eukulele_method": {
+                },
+                "eukulele_db": {
                     "type": "string",
-                    "default": "mets",
-                    "description": "Specify which method to use for EUKulele"
+                    "default": "phylodb",
+                    "description": "Specify database for EUKulele run"
```
Comment (emnilsson) [inline nextflow_schema.json]: Maybe add possible alternatives here and state default?

File: nextflow_schema.json [hunk 4]
```diff
@@ -118,74 +144,27 @@
...
-            "fa_icon": "fas fa-fast-forward",
-            "description": "Options to skip various steps within the workflow.",
-            "properties": {
-                "skip_fastqc": {
-                    "type": "boolean",
-                    "description": "Skip FastQC.",
-                    "fa_icon": "fas fa-fast-forward"
+                "hmmfiles": {
+                    "type": "string",
+                    "description": "list of hmm files to run"
```
Comment (emnilsson) [inline nextflow_schema.json:158]: Is the indentation really correct here? And below.

File: workflows/metatdenovo.nf [hunk 5]
```diff
@@ -75,7 +75,6 @@ include { MEGAHIT_INTERLEAVED              } from '../modules/local/megahit/inte
 include { HMMRANK                          } from '../modules/local/hmmrank.nf'
 include { UNPIGZ as UNPIGZ_FASTA_PROTEIN   } from '../modules/local/unpigz.nf'
 include { UNPIGZ as UNPIGZ_CONTIGS         } from '../modules/local/unpigz.nf'
-include { FORMAT_TAX                       } from '../modules/local/format_tax.nf'
```
Comment (emnilsson) [inline workflows/metatdenovo.nf:78]: Is this not needed anymore? If so, is the `format_tax.nf` deleted?

**Review by emnilsson — APPROVED** [review]
LGTM - except: in `workflows/metatdenovo.nf` on lines 134 and 135, there's an extra `/modules` for `hmmsearch` and `spades`. And some small comments, go ahead and merge when fixed if you like.

File: workflows/metatdenovo.nf (hunk 5 above)
Comment (danilodileo) [reply workflows/metatdenovo.nf:78]: It is now in subworkflow/local/eukulele.nf

**Pull request merged**